optimade/validator/utils.py

```diff
@@ -355,13 +355,16 @@ def wrapper(
             # If the result was None, return it here and ignore statuses
             if result is None:
                 return result, msg
+            else:
+                display_request = requests.utils.requote_uri(request.replace("\n", ""))  # type: ignore[union-attr]
```

Comment:
```suggestion
            display_request = requests.utils.requote_uri(request.replace("\n", ""))  # type: ignore[union-attr]
```
I think this else statement is not needed, as the code below is only executed when `result` is not none anyway.